src/main/java/io/swagger/codegen/languages/php/AbstractPhpCodegen.java

```diff
@@ -0,0 +1,667 @@
+package io.swagger.codegen.languages;
+
+import io.swagger.codegen.CliOption;
+import io.swagger.codegen.CodegenConstants;
+import io.swagger.codegen.CodegenOperation;
+import io.swagger.codegen.CodegenParameter;
+import io.swagger.codegen.CodegenProperty;
+import io.swagger.models.properties.*;
+
+import java.io.File;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.HashSet;
+import java.util.regex.Matcher;
+
+import io.swagger.v3.oas.models.media.*;
+import io.swagger.v3.parser.util.SchemaTypeUtil;
+import org.apache.commons.lang3.StringUtils;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public abstract class AbstractPhpCodegen extends DefaultCodegenConfig {
```

Comment:
`DefaultCodegenConfig` is not imported. add following line:

    import io.swagger.codegen.languages.DefaultCodegenConfig;